src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar search-bar--big"
+      action="#"
+      method="get"
+      data-qa="big"
+    >
+      <img
+        class="search-bar__img search-bar__img--big"
+        src="images/Search.svg"
+        alt="magnifying glass"
+      >
+      <input
+        class="search-bar__input search-bar__input--big"
+        data-qa="keypress"
+        type="text"
+        placeholder="Try “Los Angeles”"
+        name="searchBig"
+        >
```

Comment:
```suggestion
      <input
        class="search-bar__input search-bar__input--big"
        data-qa="keypress"
        type="text"
        placeholder="Try “Los Angeles”"
        name="searchBig"
      >
```
closing bracket should be on the same level as opening one